core-services/prow/02_config/ComplianceAsCode/compliance-operator/_prowconfig.yaml

```diff
@@ -70,6 +71,65 @@ tide:
     - needs-rebase
     repos:
     - ComplianceAsCode/compliance-operator
+  - includedBranches:
+    - main
+    - master
+    labels:
+    - approved
+    - docs-approved
+    - lgtm
+    - px-approved
+    - qe-approved
+    missingLabels:
+    - backports/unvalidated-commits
+    - bugzilla/invalid-bug
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
+    - jira/invalid-bug
+    - keep-main-query-separate
+    - needs-rebase
+    repos:
+    - ComplianceAsCode/compliance-operator
+  - excludedBranches:
+    - feature-es6x
+    - feature-prom-connector
+    - main
+    - master
+    - openshift-4.1
+    - openshift-4.2
+    - openshift-4.3
+    - openshift-4.4
+    - openshift-4.5
+    - openshift-4.6
+    - openshift-4.7
+    - openshift-4.8
+    - openshift-4.9
+    - release-3.11
+    - release-4.0
+    - release-4.1
+    - release-4.2
+    - release-4.3
+    - release-4.4
+    - release-4.5
+    - release-4.6
+    - release-4.7
+    - release-4.8
+    - release-4.9
```

Comment:
What about release-4.10? It would be cool if we could use regular expressions here.

Reply:
These usually get updated repo-wide at certain points (i.e., see #26761)

Reply:
I honestly don't know who/what adds the release branches and I don't think we use them for anything. Maybe @mrogers950  knows?

Reply:
Yeah, they don't matter for us.

Reply:
I'm fine adding them in a separate patch in this PR, though, just wouldn't want to clash with automation.